shell/platform/android/platform_view_android.cc

```diff
@@ -65,21 +66,27 @@ void PlatformViewAndroid::SurfaceCreated(JNIEnv* env,
   }
 
   ANativeWindow_release(window);
+
+  auto gl_surface = std::make_unique<GPUSurfaceGL>(surface_gl_.get());
+  NotifyCreated(std::move(gl_surface), [this, backgroundColor] {
+    rasterizer().Clear(backgroundColor, GetSize());
```

Comment:
Does this `GetSize` call actually retrieve a reasonable size?  I thought I tested this before and got 0x0 at this point.